src/lib/model/backend.ts

```diff
@@ -0,0 +1,87 @@
+export enum CriterionCategory {
+    Inclusion,
+    Exclusion,
+    HardExclusion,
+}
+
+export enum ReviewDecision {
+    No,
+    Maybe,
+    Yes,
+}
+
+export type PaperDecision = ReviewDecision.Yes | ReviewDecision.No;
+
+export interface ReviewDecisionMatrix {
+    numberOfReviewers: number;
+    patterns: Map<ReviewDecision[], PaperDecision>;
+}
+
+export interface User {
+    id: number;
+    status: string;
+    isAdmin: boolean;
+    firstName: string;
+    lastName: string;
+    email: string;
+}
+
+export type UserSpec = Omit<User, "id" | "status" | "isAdmin">;
+
+export interface Project {
+    id: number;
+    name: string;
+    reviewDecisionMatrix: ReviewDecisionMatrix;
+    similarityThreshold: number;
+    paperFetchApis: string[];
+}
+
+export type ProjectSpec = Omit<Project, "id">;
+
+export interface Criterion {
+    id: number;
+    tag: string;
+    name: string;
+    description: string;
+    category: CriterionCategory;
+}
+
+export type CriterionSpec = Omit<Criterion, "id">;
+
+export interface Review {
+    userId: number;
+    finished: boolean;
+    decision: ReviewDecision;
+    selectedCriteriaIds: number[];
+}
+
+export type ReviewSpec = Omit<Review, "userId">;
+
+export interface StageEntry {
+    paper: Paper;
+    stage: number;
+    status: string;
+    decision: PaperDecision;
+}
+
+export interface Paper {
+    id: number;
+    doi: string;
+    title: string;
+    abstrakt: string;
+    year: number;
+    type: string;
+    authorIds: number[];
+    backwardReferencedPaperIds: number[];
```

Comment:
For the full list of references, we would also need the forward referenced paper IDs, can we do that?

Reply:
I initially intended forward references to be accessed solely through `getForwardReferencedPapers` of a paper as they are a direct computational result of backward references. On second thought it would be a nice API addition to just put them in a project. This would ease the addition of forward references without having to go to the referencing paper.

Will be adding this.

Reply:
Great!